doc/contributors/README.md

```diff
@@ -0,0 +1,38 @@
+# Contributors-oriented documentation
+
+## Where to Start
+
+Start with [Repo Structure and Tooling](./contributors/structure.md).
+
+### Index
+
+- **Conventions**
+  - [Repo Structure and Tooling](./structure.md)
+    - How directories and files are organized in our GitHub repo
+    - What tools are used to build parts of Puter
+  - [Comment Prefixes](./comment_prefixes.md)
+    - A convention we use for line comments in code
+
+- [Frontend Documentation](../../src/gui/doc/)
+- [Backend Documentation](../../src/backend/doc/README.md)
+- [Extensions](./extensions/README.md)
+
+## How to find documentation on something 
+
+You will be able to find existing documentation on a contribution topic **following the structure** of the contributor documentation.
+[`This is the structure`](./structure.md) of puter's contributors-oriented docs.
+
+## How to understand the code and work of others more easily
+
+**Comments** are used in code to make them easier to read.
```

Comment:
The statement about what comments are for is unnecessary, but this would be a good place to describe effective ways to use comments.

```suggestion
**Comments** should supplement the code. Instead of describing what the code does, describe the code's purpose or contextual information that makes it easier to understand.
```